docs/en/stack/ml/anomaly-detection/index.asciidoc

```diff
@@ -4,37 +4,30 @@
 
 [partintro]
 --
-Machine learning is tightly integrated with the {stack}. Data is pulled
-from {es} for analysis and anomaly results are displayed in {kib} dashboards.
+{ml-cap} is tightly integrated with the {stack}. Data is pulled from {es} for 
+analysis and anomaly results are displayed in {kib} dashboards.
 
 * <<ml-overview>>
-* <<create-jobs>>
+* <<ml-concepts>>
+* <<ml-configuration>>
 //* <<ml-getting-started>>
-* <<ml-configuring>>
-* <<stopping-ml>>
-* <<ml-troubleshooting, Troubleshooting Machine Learning>>
-* <<ml-api-quickref>>
-* <<ml-functions>>
+* <<ml-reference>>
+* <<ml-limitations>>
+//* <<ml-troubleshooting>>
```

Comment:
What's the plan for relocating this troubleshooting content?

Reply:
@lcawl AFAIK, the content under troubleshooting is not valid anymore. I'd like to double-check them and delete the irrelevant ones from the master/7.x/7.4 branches.